app/uk/gov/hmrc/alcoholdutyreturns/models/audit/AuditReturnSubmitted.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutyreturns.models.audit
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.alcoholdutyreturns.models.AlcoholRegime
+import uk.gov.hmrc.alcoholdutyreturns.models.audit.AuditType.ReturnSubmitted
+import uk.gov.hmrc.alcoholdutyreturns.models.returns.{AdrReturnCreatedDetails, ReturnCreate}
+
+import java.time.Instant
+
+case class AuditReturnSubmitted(
+  appaId: String,
+  periodKey: String,
+  governmentGatewayId: Option[String],
+  governmentGatewayGroupId: Option[String],
+  returnSubmittedTime: Instant,
+  alcoholRegimes: Option[Set[AlcoholRegime]],
```

Comment:
why is this one optional ? There must be at least one regime otherwise you get a something went wrong due to an exception throw.